diff -up ckb-0.2.5/src/ckb/ckb.pro.external-quazip ckb-0.2.5/src/ckb/ckb.pro
--- ckb-0.2.5/src/ckb/ckb.pro.external-quazip	2016-04-13 23:21:18.622742135 +0200
+++ ckb-0.2.5/src/ckb/ckb.pro	2016-04-13 23:45:15.507683725 +0200
@@ -31,8 +31,7 @@ CKB_VERSION_STR = `cat $$PWD/../../VERSI
 DEFINES += CKB_VERSION_STR="\\\"$$CKB_VERSION_STR\\\""
 
 # Zip library for decompressing firmwares
-LIBS += -lz
-DEFINES += QUAZIP_STATIC
+LIBS += -lquazip5 -lz
 
 linux {
 # Conditionally use libappindicator to support Unity indicators
@@ -75,19 +74,6 @@ SOURCES += main.cpp\
     kbbind.cpp \
     rebindwidget.cpp \
     modeselectdialog.cpp \
-    quazip/JlCompress.cpp \
-    quazip/qioapi.cpp \
-    quazip/quaadler32.cpp \
-    quazip/quacrc32.cpp \
-    quazip/quagzipfile.cpp \
-    quazip/quaziodevice.cpp \
-    quazip/quazip.cpp \
-    quazip/quazipdir.cpp \
-    quazip/quazipfile.cpp \
-    quazip/quazipfileinfo.cpp \
-    quazip/quazipnewinfo.cpp \
-    quazip/unzip.c \
-    quazip/zip.c \
     kbfirmware.cpp \
     fwupgradedialog.cpp \
     autorun.cpp \
@@ -129,22 +115,6 @@ HEADERS  += mainwindow.h \
     kbbind.h \
     rebindwidget.h \
     modeselectdialog.h \
-    quazip/crypt.h \
-    quazip/ioapi.h \
-    quazip/JlCompress.h \
-    quazip/quaadler32.h \
-    quazip/quachecksum32.h \
-    quazip/quacrc32.h \
-    quazip/quagzipfile.h \
-    quazip/quaziodevice.h \
-    quazip/quazip.h \
-    quazip/quazip_global.h \
-    quazip/quazipdir.h \
-    quazip/quazipfile.h \
-    quazip/quazipfileinfo.h \
-    quazip/quazipnewinfo.h \
-    quazip/unzip.h \
-    quazip/zip.h \
     kbfirmware.h \
     fwupgradedialog.h \
     autorun.h \
diff -up ckb-0.2.5/src/ckb/kbfirmware.cpp.external-quazip ckb-0.2.5/src/ckb/kbfirmware.cpp
--- ckb-0.2.5/src/ckb/kbfirmware.cpp.external-quazip	2016-04-13 23:33:43.308125528 +0200
+++ ckb-0.2.5/src/ckb/kbfirmware.cpp	2016-04-13 23:34:08.119825371 +0200
@@ -1,7 +1,7 @@
 #include "kbfirmware.h"
 #include "kbmanager.h"
-#include "quazip/quazip.h"
-#include "quazip/quazipfile.h"
+#include "quazip5/quazip.h"
+#include "quazip5/quazipfile.h"
 #include <QDateTime>
 #include <QDebug>
