diff -up ckb-0.2.5/src/ckb-daemon/devnode.c.ckb-use-var-run ckb-0.2.5/src/ckb-daemon/devnode.c
--- ckb-0.2.5/src/ckb-daemon/devnode.c.ckb-use-var-run	2016-04-15 18:31:42.279323827 +0200
+++ ckb-0.2.5/src/ckb-daemon/devnode.c	2016-04-15 18:32:11.795435494 +0200
@@ -6,12 +6,7 @@
 #include "notify.h"
 #include "profile.h"
 
-// OSX doesn't like putting FIFOs in /dev for some reason
-#ifndef OS_MAC
-const char *const devpath = "/dev/input/ckb";
-#else
 const char *const devpath = "/var/run/ckb";
-#endif
 
 long gid = -1;
 #define S_GID_READ  (gid >= 0 ? S_CUSTOM_R : S_READ)
diff -up ckb-0.2.5/src/ckb/kbmanager.cpp.ckb-use-var-run ckb-0.2.5/src/ckb/kbmanager.cpp
--- ckb-0.2.5/src/ckb/kbmanager.cpp.ckb-use-var-run	2016-04-15 18:32:38.431548333 +0200
+++ ckb-0.2.5/src/ckb/kbmanager.cpp	2016-04-15 18:32:57.043631740 +0200
@@ -1,10 +1,6 @@
 #include "kbmanager.h"
 
-#ifndef Q_OS_MACX
-QString devpath = "/dev/input/ckb%1";
-#else
 QString devpath = "/var/run/ckb%1";
-#endif
 
 QString KbManager::_guiVersion, KbManager::_daemonVersion = DAEMON_UNAVAILABLE_STR;
 KbManager* KbManager::_kbManager = 0;

